### PR TITLE
Add version history for 5.1.9

### PR DESCRIPTION
**Pull request by bramalingam**
Preliminary draft for the version history. Please feel free to point out edits/corrections.

@melissalinkert @jburel @sbesson @joshmoore

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+       - fixed the parsing of emission wavelength.
+   - APL
+       - fixed multiple parsing issues with the mtb file.
+   - SlideBook
+       - removed slidebook dlls from Bio-Formats repository.
+   - Zeiss CZI
+       - fixed parsing of files with multiple mosaics and positions.
+
+* Documentation updates, including:
+   - improved documentation for the export of big-tiff’s in ImageJ.
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: big-tiffs doesn't need an apostrophe and should be written as `BigTIFFs` to match elsewhere in the docs

**Comment by hflynn** [comment]
Previous entries don't have periods at the end of any of the bullet points, please remove these for consistency to match our style guide

File: docs/sphinx/about/whats-new.txt [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+       - fixed width padding calculation for single-pixel image.
+   - Deltavision
+       - fixed the parsing of the new date format.
+       - added support for parsing and storing the working distance in native units.
+   - Micromanager
+       - cleaned up JSON metadata parsing.
+   - Olympus Fluoview
+       - fixed null pointer exceptions while parsing metadata.
+   - Leica LIF
+       - fixed large milt-tile files from having incorrect plane offsets after the 2GB mark.
```
Comment (melissalinkert) [inline docs/sphinx/about/whats-new.txt]: ```milt-tile``` => ```multi-tile``` or ```tiled```

File: docs/sphinx/about/whats-new.txt [hunk 3]
```diff
@@ -1,6 +1,45 @@
...
+   - EM formats
```
Comment (melissalinkert) [inline docs/sphinx/about/whats-new.txt]: Clarify that this is MRC and Spider specifically (per https://github.com/openmicroscopy/bioformats/pull/2298/files)?

File: docs/sphinx/about/whats-new.txt [hunk 4]
```diff
@@ -1,6 +1,45 @@
...
+       - added native length support for EM readers.
+   - Gatan
+       - fixed erroneous metadata parsing.
+       - added support for parsing and storing the physical sizes in native units.
+   - OME-TIFF
+       - improve handling of OME-TIFF multi-file file-sets with partial metadata blocks.
+   - Nikon ND2
+       - fixed the parsing of emission wavelength.
+   - APL
```
Comment (melissalinkert) [inline docs/sphinx/about/whats-new.txt]: Usually noted as ```Olympus CellR (APL)``` or ```Olympus CellR/APL``` (for clarity, since the reader/instrument name is CellR).

**Comment by melissalinkert** [comment]
Not sure if it's worth saying anything about https://github.com/openmicroscopy/bioformats/pull/2300 (kind of defer to @joshmoore).

**Comment by joshmoore** [comment]
Big +1 for including gh-2300

File: docs/sphinx/about/whats-new.txt [hunk 5]
```diff
@@ -1,6 +1,45 @@
...
+       - removed slidebook dlls from Bio-Formats repository
+   - Zeiss CZI
+       - fixed parsing of files with multiple mosaics and positions
+
+* Documentation updates, including:
+   - improved documentation for the export of BigTIFFs in ImageJ
+
+* C++:
+   - xerces version 3.1.2 is now removed from the mirror
+   - xerces version updated to Version 3.1.3
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: This is unrelated to the Bio-Formats repository where no C++ change happened so I would not mention it in the history.

Comment (bramalingam) [inline docs/sphinx/about/whats-new.txt]: So should I remove the C++ part, 

or just mention 

* C++:
- no changes

Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: C++ no change would be sufficient as we have done before

File: docs/sphinx/about/whats-new.txt [hunk 6]
```diff
@@ -1,6 +1,44 @@
...
+       - fixed null pointer exceptions while parsing metadata
+   - Leica LIF
+       - fixed large multi-tiled files from having incorrect plane offsets after the 2GB mark
+   - EM formats (MRC and Spider)
+       - added native length support for EM readers
+   - Gatan
+       - fixed erroneous metadata parsing
+       - added support for parsing and storing the physical sizes in native units
+   - OME-TIFF
+       - improve handling of OME-TIFF multi-file file-sets with partial metadata blocks
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: All the other points are past tense, this should be 'improved' to match. Also 'fileset' is written as one word in the docs

File: docs/sphinx/about/whats-new.txt [hunk 7]
```diff
@@ -1,6 +1,44 @@
...
+       - cleaned up JSON metadata parsing
+   - Olympus Fluoview
+       - fixed null pointer exceptions while parsing metadata
+   - Leica LIF
+       - fixed large multi-tiled files from having incorrect plane offsets after the 2GB mark
+   - EM formats (MRC and Spider)
+       - added native length support for EM readers
+   - Gatan
+       - fixed erroneous metadata parsing
+       - added support for parsing and storing the physical sizes in native units
```
Comment (sbesson) [inline docs/sphinx/about/whats-new.txt]: Note this item could be grouped under the previous bullet point describing support for native physical size unit across EM formats (since `Gatan` is one of them). Leaving it as part of a `Gatan` bullet point works as well.

File: docs/sphinx/about/whats-new.txt [hunk 8]
```diff
@@ -1,6 +1,44 @@
...
+   - OME-TIFF
+       - improved handling of OME-TIFF multi-file fileset’s with partial metadata blocks
+   - Nikon ND2
+       - fixed the parsing of emission wavelength
+   - Olympus CellR (APL)
+       - fixed multiple parsing issues with the mtb file
+   - SlideBook
+       - removed slidebook dlls from Bio-Formats repository
```
Comment (joshmoore) [inline docs/sphinx/about/whats-new.txt]: Once ready, I'd include a doc URL here to tell users that they will need to activate the update site

File: docs/sphinx/about/whats-new.txt [hunk 9]
```diff
@@ -1,6 +1,44 @@
 Version history
 ===============
 
+5.1.9 (2016 April 11)
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: The date is going to need updating but otherwise this looks ok to me now. Given you already have a separate bullet point for Gatan, I'd be inclined to leave the item @sbesson commented on where it is.

File: docs/sphinx/about/whats-new.txt (hunk 8 above)
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: URL to add here is http://www.openmicroscopy.org/info/slidebook

**Comment by hflynn** [comment]
Rebuilding staging now

**Comment by hflynn** [comment]
Good to merge

**Comment by sbesson** [comment]
--rebased-to #2355
